db/migrate/20200914221259_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
```

Comment:
Looks like we want to allow users to sign-in by using their employee-number
Any plans for adding that column in this table? 😄 

Can we confirm if the email is needed?
I mean, it'd be useful to have it since it'd help with the reset-password functionality